src/App.js

```diff
@@ -396,9 +370,7 @@ function Root() {
 }
 
 const RootWithSentry = Sentry.wrap(Root);
-const RootWithCodePush = codePush({
-  checkFrequency: codePush.CheckFrequency.MANUAL,
-})(RootWithSentry);
+const RootWithCodePush = codePush(RootWithSentry);
```

Comment:
Do we still need `codePush` here?

Reply:
This feature used codepush, but we use codepush separately as well. We haven't used that in a long time so generally might be worth checking if we actually still need it, but yeah it is also configured for OTA updates